brainstormforce/astra
# AST-764 - Editor lists blocks outside screen intent issue fixed

### Description
- List marks shows outside editor contained
- For frontend marks issue as below please use this PR - https://github.com/brainstormforce/astra/pull/3110
- Astra - https://share.getcloudapp.com/xQu7BKqr
- 2020 - https://share.getcloudapp.com/mXur0dJZ

### Screenshots
- Issue - https://share.bsf.io/P8uGYzZ6
- With fix - https://share.bsf.io/bLuqG2z9

### Types of changes
- Non-breaking change 

### How has this been tested?
- Check with lists blocks (with alternate list styles)
- Try by checking adding lists blocks in nested block
- Check if builder lists ul/ol affects with `margin-left: 20px;` CSS

### Checklist:
